dev/scripts/oci_replication_func.py

```diff
@@ -75,23 +80,29 @@ def copy_reports_to_bucket(
     last_reports,
 ):
     # Iterate through cost reports list and copy them to new bucket
+    # Get todays datetime
+    today = datetime.datetime.now()
+    # Start from current month
+    start_from = today.replace(day=1)
```

Comment:
suggestion: This is the only block where you use `today`. Perhaps this would be better:
```suggestion
    # Start from current month
    start_from = datetime.date.today().replace(day=1)
```